src/main/java/org/jitsi/protocol/xmpp/AbstractOperationSetJingle.java

```diff
@@ -103,12 +104,38 @@ public boolean initiateSession(boolean useBundle,
 
         sessions.put(sid, session);
 
+        JingleIQ inviteIQ = createInviteIQ(
+                sid, useBundle, address, contents, startMuted);
```

Comment:
I'm not sure why the above formatting is preferred over:

JingleIQ inviteIQ
    = createInviteIQ(sid, useBundle, address, contents, startMuted);

I find my example to be easier to read.